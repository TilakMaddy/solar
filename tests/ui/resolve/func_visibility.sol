contract U1 {
    function c() {} //~ERROR: No visibility specified. Did you intend to add public?
}

interface U2 {
    function c() {} //~ERROR: No visibility specified. Did you intend to add external?
}

contract U3 { //~ERROR: No visibility specified for fallback. Did you intend to add external?
    fallback() {}
}

contract U4 {//~ERROR: No visibility specified for receive. Did you intend to add external?
    receive() {}
}
