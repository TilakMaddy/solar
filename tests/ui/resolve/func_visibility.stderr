error: No visibility specified. Did you intend to add public?
  --> ROOT/tests/ui/resolve/func_visibility.sol:LL:CC
   |
LL |     function c() {}
   |              ^
   |

error: No visibility specified. Did you intend to add external?
  --> ROOT/tests/ui/resolve/func_visibility.sol:LL:CC
   |
LL |     function c() {}
   |              ^
   |

error: No visibility specified for fallback. Did you intend to add external?
  --> ROOT/tests/ui/resolve/func_visibility.sol:LL:CC
   |
LL | contract U3 {
   |          ^^
   |

error: No visibility specified for receive. Did you intend to add external?
  --> ROOT/tests/ui/resolve/func_visibility.sol:LL:CC
   |
LL | contract U4 {
   |          ^^
   |

error: aborting due to 4 previous errors

